doc/tests/tests_tutorial.rst

```diff
@@ -71,7 +71,7 @@ Test coverage measures the lines of code that are executed while running the tes
 To accumulate statistics and create a html coverage report, build the code without optimization
 and run the special `_coverage` target::
 
-  sudo apt-get install ros-melodic-code-coverage
+  sudo apt-get install ros-noetic-code-coverage
```

Comment:
Prob should change this to `sudo apt install`

Reply:
I thought about making that change.